core-it-suite/src/test/resources/mng-7566/repo/org/apache/maven/its/mng7566/maven-mng7566-plugin/0.1/maven-mng7566-plugin-0.1.pom

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>org.apache.maven.its.mng7566</groupId>
+  <artifactId>maven-mng7566-plugin</artifactId>
+  <version>0.1</version>
+  <packaging>maven-plugin</packaging>
+
+  <name>Maven Integration Test Plugin :: Touch</name>
+
+  <distributionManagement>
+    <repository>
+      <id>maven-core-it</id>
+      <url>file:///${basedir}/repo</url>
+    </repository>
+  </distributionManagement>
+
+  <properties>
+    <maven.test.skip>true</maven.test.skip>
+  </properties>
+
+  <dependencies>
+    <dependency>
+      <groupId>org.apache.maven</groupId>
+      <artifactId>maven-plugin-api</artifactId>
+      <version>2.0</version>
```

Comment:
Don't use stoneage deps.

Reply:
This is just a copy from MavenITmng4840MavenPrerequisiteTest. Why does it matter here? Which version do you suggest?

Reply:
I bet that @gnodet has already updated this in his branch. The reason is to reduce variance in dependencies. Maven 2 is dead, therefore we should avoid using those artifacts even if the API hasn't changed.

Reply:
In master it still is 2.0: https://github.com/apache/maven-integration-testing/blob/bd7ba0adf5f847b394330cfc022ea833e022ebe5/core-it-suite/src/test/resources/mng-4840/repo/org/apache/maven/its/mng4840/maven-mng4840-plugin/0.1/maven-mng4840-plugin-0.1.pom#L51.
But I can change to 3.8.6.

Reply:
Let's leave it like this. This can been sweeped all at once.

Reply:
https://github.com/apache/maven-integration-testing/blob/582dd0199262595fb647eb34c05d60aa4ad86941/core-it-suite/src/test/resources/mng-4840/repo/org/apache/maven/its/mng4840/maven-mng4840-plugin/0.1/maven-mng4840-plugin-0.1.pom#L51